dev-guide/kubernetes-continuous-rebase.md

```diff
@@ -0,0 +1,133 @@
+---
+title: kubernetes-rebase
+authors:
+  - "@fbertina"
+reviewers:
+  - "@soltysh"
+approvers:
+  - "@soltysh"
+creation-date: 2023-06-15
+last-updated: 2023-06-15
+---
+
+# Kubernetes Continuous Rebase
```

Comment:
@soltysh I'm calling it "continuous" rather than "automated" for now because we're not there yet :smile:. Also, most likely it will always required manual intervention for fixing the issues.